src/main/java/com/webperside/courseerpbackend/controller/GroupController.java

```diff
@@ -0,0 +1,31 @@
+package com.webperside.courseerpbackend.controller;
+
+import com.webperside.courseerpbackend.models.base.BaseResponse;
+import com.webperside.courseerpbackend.models.mappers.GroupEntityMapper;
+import com.webperside.courseerpbackend.models.payload.group.GroupPayload;
+import com.webperside.courseerpbackend.services.group.GroupService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/groups")
+@RequiredArgsConstructor
+public class GroupController {
+
+    private final GroupService groupService;
+    private static final GroupEntityMapper groupEntityMapper =GroupEntityMapper.INSTANCE;
```

Comment:
You don't need to store INSTANCE value of Mapper separately. it is unnecessary. use it directly . f.e
GroupEntityMapper.INSTANCE.toEntity()